azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -44,13 +44,14 @@ def __init__(self, iothub_pipeline):
         self._edge_pipeline = None
 
     @classmethod
-    def create_from_connection_string(cls, connection_string, ca_cert=None):
+    def create_from_connection_string(cls, connection_string, ca_cert=None, transport=None):
```

Comment:
>None [](start = 86, length = 4)

If the default really is mqtt, then you should have "transport=mqtt" in the function definition.